lang/nim/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                nim
+version             1.4.4
+revision            0
+license             MIT
+categories          lang
+platforms           darwin
+supported_archs     i386 x86_64 ppc64
+maintainers         {@gpanders gpanders.com:greg} \
+                    openmaintainer
+
+description         Nim programming language
+long_description    Nim is a statically typed compiled systems programming \
+                    language. It combines successful concepts from mature \
+                    languages like Python, Ada, and Modula. Its design focuses \
+                    on efficiency, expressiveness, and elegance (in that \
+                    order of priority).
+
+homepage            https://nim-lang.org
+
+master_sites        https://nim-lang.org/download/
+
+checksums           rmd160  df2568109792112495aaed4273f222bf8623f95b \
+                    sha256  9f43f5da8f7130b137d59e49f3911f0c35cc1e94e68c333c50c59ef8b65dd6f6 \
+                    size    159095112
+
+use_configure       no
+
+variant doc description {Build HTML docs} {}
+
+build {
+    system -W ${worksrcpath} "./build.sh --os ${os.platform} --cpu ${os.arch}"
```

Comment:
Per my current understanding, you may want to use `${configure.build_arch}` here.

See https://github.com/macports/macports-ports/commit/84dd03127da4647acd1fea8d612dedde07c7e3f3

Reply:
Actually I may be wrong on this one, disregard.